src/Config/ObjectTypeDefinition.php

```diff
@@ -58,7 +65,6 @@ private function treatFieldsDefaultAccess(ArrayNodeDefinition $node): void
                     if (array_key_exists('access', $field) && null !== $field['access']) {
```

Comment:
```suggestion
                    if (isset($field['access'])) {
```
The isset operator checks, if the key exists and is not null